nesh/02_customize_your_cluster.ipynb

```diff
@@ -0,0 +1,364 @@
+      "       12    167.855        20     22.855 neshcl280       clmedium       \n",
+      "       12    168.238        20     22.473 neshcl293       clmedium       \n",
+      "       12    168.281        20     22.430 neshcl257       clmedium       \n",
+      "       12    170.238        20     20.473 neshcl230       clmedium       \n",
+      "       12    170.602        20     20.109 neshcl253       clmedium       \n",
+      "       12    170.672        20     20.039 neshcl332       clmedium       \n",
+      "       12    170.727        20     19.984 neshcl305       clmedium       \n",
+      "       12    171.816        20     18.895 neshcl277       clmedium       \n",
+      "       14    169.398        18     21.312 neshcl310       clmedium       \n",
+      "       22    169.422        10     21.289 neshcl279       clmedium       \n",
+      "       32    173.406         0     17.305 neshcl231       clmedium       \n",
+      "       32    188.000         0      2.711 neshcl313       clmedium       \n",
+      "        4      4.082        28    186.629 neshcl341       clmedium       \n",
+      "        5    122.023        27     68.688 neshcl301       clmedium       \n",
+      "        6      3.613        26    187.098 neshcl287       clmedium       \n",
+      "        6      4.855        26    185.855 neshcl299       clmedium       \n",
+      "        6     98.504        26     92.207 neshcl284       clmedium       \n",
+      "        7    100.797        25     89.914 neshcl292       clmedium       \n",
+      "        7      3.625        25    187.086 neshcl250       clmedium       \n",
+      "        7     75.750        25    114.961 neshcl294       clmedium       \n",
+      "        8    139.859        24     50.852 neshcl334       clmedium       \n",
+      "        8      3.680        24    187.031 neshcl262       clmedium       \n",
+      "       32    184.520         0      6.191 neshcl343       cltestque      \n",
+      "       32    179.602         0     11.109 neshcl342       cltestque,clinteractive\n"
+     ]
+    }
+   ],
+   "source": [
+    "%%bash\n",
+    "qstat -E -F fcpu,fmem1,ucpu,umem1,ehost,quenm | head -n2 # provide header\n",
+    "#qstat -E -F fcpu,fmem1,ucpu,umem1,ehost,quenm | awk '{ if ($1>0 && substr($5,1,6)==\"neshcl\") { print } }' | sort -k6 # raw output\n",
+    "qstat -E -F fcpu,fmem1,ucpu,umem1,ehost,quenm | awk '{ if ($1>0 && substr($5,1,6)==\"neshcl\") { printf \"%9d%11.3f%10d%11.3f %-15s %-15s\\n\",$1,$2/2**8,$3,$4/2**8,$5,$6 } }' | sort -k6 # memory units in GB"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Start Dask jobqueue cluster in a different batch class\n",
+    "Manually choosing a batch queue for your Dask jobqueue cluster is as simple as using the default configuration and calling the `PBSCluster` method with the `queue` keyword argument. This simply alters the PBS header specifying the batch class in the Dask worker job script."
```

Comment:
> Manually choosing a batch queue for your Dask jobqueue cluster is as simple as using the default configuration and calling the `PBSCluster` method with the `queue` keyword argument. This simply alters the PBS header specifying the batch class in the Dask worker job script.

-> Manually choosing a batch queue for your Dask jobqueue cluster is as simple as using the default configuration and calling the `PBSCluster` method with the `queue` keyword argument **alone**. This will override the selection of the queue, but will follow all other aspects of the default config.